requirements.txt

```diff
@@ -5,4 +5,4 @@ drf-nested-routers~=0.93.3
 drf-yasg~=1.20.0
 django-autocomplete-light~=3.8.2
 ensembl-prodinf-djcore@git+https://github.com/Ensembl/ensembl-prodinf-djcore.git@1.1#egg=ensembl-prodinf-djcore
-ensembl-prodinf-core@git+https://github.com/Ensembl/ensembl-prodinf-core.git@1.1#egg=ensembl-prodinf-core
+ensembl-prodinf-core@git+https://github.com/Ensembl/ensembl-prodinf-core.git@1.3#egg=ensembl-prodinf-core
```

Comment:
I presume the dependency could be change to 
ensembl-prodinf-core~=2.0.0 ?
Assuming this is the corresponding version of https://github.com/Ensembl/ensembl-prodinf-core/pull/20 I just validated.

Reply:
Correct. I'll change it as soon as we have core libs on PyPI.